### PR TITLE
Added instructions I needed to install gendoc

### PR DESCRIPTION
**Pull request by johndmulhausen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,11 +14,26 @@ then open a terminal and run the following (example for OS X):
 ```
 $ cd Downloads
 $ chmod 755 gendoc-osx
+$ mkdir /usr/local/bin/gendoc
```
Comment (SvenDowideit) [inline README.md:17]: this makes no sense - `/usr/local/bin/gendoc` is the destination filename where the executable file (called `gendoc-osx` on github) should go

**Comment by SvenDowideit** [comment]
If you follow the instructions as they are, the `./gendoc-osx install` command will create a file called `/usr/local/bin/gendoc` which will be in your path, and will run.

if those instructions don't work - then please report that with more info - I've not seen it fail yet, so i'm curious what prompted this PR.

File: README.md (hunk 1 above)
Comment (johndmulhausen) [inline README.md:17]: Try that on a new system. You will get a "directory does not exist" error. (Or at least, I did)

Comment (SvenDowideit) [inline README.md:17]: so your system didn't have `/usr/local/bin` ?

**Pull request closed without merging**